src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* Updating ELFSharp Package to version 2.14.0. [#628](https://github.com/microsoft/binskim/pull/628)
```

Comment:
>Updating [](http://example.com/codeflow?start=2&length=8)

You did not mention the previous version. Just use the PR title, it's descriptive and concise.

'Bump ELFSharp from 2.13.2 to 2.14.0.

Reply:
Also you have capitalize 'Package' unexpected.

Beware strange Capitalizations in Text, this generally isn't Correct.

Reply:
Thanks!
Just updated